tests/bluetooth/bsim_bt/bsim_test_notify/test_scripts/multi_notify.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Basic GATT test: A central acting as a GATT client scans for and connects
+# to a peripheral acting as a GATT server. The GATT client will then attempt
+# to write and read to and from a few GATT characteristics.
+simulation_id="multi_notify"
+verbosity_level=2
+process_ids=""; exit_code=0
+
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "  \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  timeout 120 $@ & process_ids="$process_ids $!"
```

Comment:
`120` is a long timeout for such as simple test. Suggest something shorter like 10 or 20 seconds.